ssclj/jar/test/com/sixsq/slipstream/ssclj/resources/session_oidc/utils_test.clj

```diff
@@ -18,6 +18,19 @@
   (is (= ["a:a"] (t/extract-roles {:realm "a" :roles ["a"]})))
   (is (= ["a:a" "a:b"] (t/extract-roles {:realm "a" :roles ["a" "b"]}))))
 
+(deftest check-extract-entitlements
+  (is (= [] (t/extract-entitlements nil)))
+  (is (= [] (t/extract-entitlements {})))
+  (is (= [] (t/extract-entitlements {:realm "a"})))
+  (is (= [] (t/extract-entitlements {:entitlement []})))
+  (is (= [] (t/extract-entitlements {:realm "a" :entitlement []})))
+  (is (= ["a:a"] (t/extract-entitlements {:realm "a" :entitlement "a"})))
+  (is (= ["a:alpha"] (t/extract-entitlements {:realm "a" :entitlement "alpha"})))
+  (is (= ["a:a"] (t/extract-entitlements {:realm "a" :entitlement ["a"]})))
+  (is (= ["a:alpha"] (t/extract-entitlements {:realm "a" :entitlement ["alpha"]})))
+  (is (= ["a:a" "a:b"] (t/extract-entitlements {:realm "a" :entitlement ["a" "b"]})))
+  (is (= ["a:alpha" "a:beta"] (t/extract-entitlements {:realm "a" :entitlement ["alpha" "beta"]}))))
```

Comment:
I tried adding extra tests 
* Some of those extra tests are passing : 
```
(is (= [] (t/extract-entitlements {:entitlement nil})))
(is (= [] (t/extract-entitlements {:entitlement ""})))
(is (= [] (t/extract-entitlements {:realm "a" :entitlement nil})))
(is (= ["a:a" "a:b" "a:c"] (t/extract-entitlements {:realm "a" :entitlement ["a" "b" "c"]})))
```
* Some other don't
```
;;empty realm
(is (= [] (t/extract-entitlements {:realm "" :entitlement "a"})))
;;not sure if uniqueness of entitlements is relevant
(is (= ["a:a" "a:b"] (t/extract-entitlements {:realm "a" :entitlement ["a" "b" "a"]})))
;; not sure is case sensitivity and uniqueness has to be considered
(is (= ["a:alpha" "a:beta"] (t/extract-entitlements {:realm "a" :entitlement ["alpha" "beta" "Alpha" "BETA"]})))
;;entitlement is an instance of String, but is empty
(is (= [] (t/extract-entitlements {:realm "a" :entitlement ""})))
(is (= [] (t/extract-entitlements {:realm "a" :entitlement [""]})))
(is (= [] (t/extract-entitlements {:realm "a" :entitlement [nil]})))
(is (= ["a:a" "a:b"] (t/extract-entitlements {:realm "a" :entitlement ["a" "b" nil]})))
(is (= ["a:a" "a:b"] (t/extract-entitlements {:realm "a" :entitlement ["a" "b" ""]})))
```